rust/repositories.bzl

```diff
@@ -589,8 +626,10 @@ def rust_repository_set(
 
     Args:
         name (str): The name of the generated repository
-        version (str): The version of the tool among "nightly", "beta', or an exact version.
         exec_triple (str): The Rust-style target that this compiler runs on
+        version (str): The version of the tool among "nightly", "beta', or an exact version.
```

Comment:
nit: replace `"beta'` with `"beta"`; also add an example for how an exact version looks like, like the `1.65.0` above.